cmd/create/machinepool/cmd.go

```diff
@@ -169,6 +170,14 @@ func init() {
 		"",
 		"Select subnet to create a single AZ machine pool for BYOVPC cluster")
 
+	flags.StringVar(
+		&args.version,
+		"version",
+		"",
+		"Version of OpenShift that will be used to install a machine pool for a hosted cluster,"+
```

Comment:
np: Openshift Version that will be used to install a machine pool for a hosted cluster